CMakeLists.txt

```diff
@@ -143,7 +143,7 @@ add_library(dronecore ${library_type}
 target_link_libraries(dronecore
     ${CMAKE_THREAD_LIBS_INIT}
     ${curl_lib}
-    tinyxml2
+#    tinyxml2
```

Comment:
Please don't change this stuff. Make sure to either merge or rebase on current develop and get rid of this diff.

Reply:
I'd say rebase